### DISCUSSION TITLE
Is it now safe to say that flet is production ready?

### DISCUSSION THREAD

**Discussion by TobiasHT5** [flet-dev/flet#3439] [General]
### Question

I have been wondering if it's safe to say that flet is production ready and is safe to use in consumer products. Given the fact that some widgets like the bottomsheet and audio still have issues, I don't know if it's safe to call it production ready. What does the team think?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Flet is still `pre-v1.0`, so not officially stable.
